prisma/schema.prisma

```diff
@@ -0,0 +1,18 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DB_LOCAL_URL")
+}
+
+model User {
+  id        String   @id @default(uuid())
+  name      String
+  email     String   @unique
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now())
```

Comment:
o correto é utiliza updatedAt do ORM: https://www.prisma.io/docs/orm/more/migrating-to-prisma/migrate-from-sequelize#timestamps